llm_on_ray/inference/models/gpt-j-6b.yaml

```diff
@@ -14,17 +14,4 @@ ipex:
 model_description:
   model_id_or_path: EleutherAI/gpt-j-6b
   tokenizer_name_or_path: EleutherAI/gpt-j-6b
-  chat_processor: ChatModelGptJ
   gpt_base_model: true
-  prompt:
-    intro: 'Below is an instruction that describes a task. Write a response that appropriately
-      completes the request.
-
-      '
-    human_id: '
-
-      ### Instruction'
-    bot_id: '
-
-      ### Response'
-    stop_words: []
```

Comment:
Does gpt-j-6b have a default chat template in transformers? If not, please add chat_template key for it. This is also required for other models' configuration file.

Reply:
Our default chat template is in gpt-j-6b format.